src/main/java/jenkins/plugins/logstash/persistence/LogstashIndexerDao.java

```diff
@@ -39,7 +39,8 @@
   static enum IndexerType {
     REDIS,
     RABBIT_MQ,
-    ELASTICSEARCH
+    ELASTICSEARCH,
+    JMS
```

Comment:
Maybe call it "ACTIVE_MQ" instead? These values are used to populate the dropdown list in the global configuration.